src/lib/toolbar.ts

```diff
@@ -180,9 +180,14 @@ export const toolbarConfig: Record<ToolbarModeName, ToolbarMode> = {
       },
       {
         id: 'chamfer',
-        onClick: () => console.error('Chamfer not yet implemented'),
+        onClick: ({ commandBarSend }) =>
+          commandBarSend({
+            type: 'Find and select command',
+            data: { name: 'Chamfer', groupId: 'modeling' },
+          }),
         icon: 'chamfer3d',
-        status: 'kcl-only',
+        status: DEV ? 'available' : 'kcl-only',
```

Comment:
(Not specific to this PR, no change requested here)

@franknoirot I think we might want to start checking against another variable than DEV here? Something that could resolve to true on app.zoo.dev and nightly builds, but not release builds. Seems to me that DEV might be a tad too restrictive here. But I could be wrong too.

Reply:
Nice! I’d be happy to follow the same path for `fillets` and `chamfers` while waiting for the **engine team** to unblock them for `revolves`.  Here’s the video showing how it works now: #4694.

Overall, I think shipping **semi-working features** for **nightly builds** could be a good practice, as we have enough warning messages in place. Including the Command Bar UI warnings. We could add an extra indication for the buttons that are "nightly only". 

Besides, DEV blocks part of our team like Nick and Josh from checking and testing new stuff. 

On the other hand, it doesn’t fully align with the general rule: _“Don’t ship shit.”_  : D 

@jessfraz,  @Irev-Dev , what do you think?

Reply:
I like this idea, we should add another environment variable for this. @pierremtb would you be able to do that?

Reply:
For sure! And should be a separate PR too

Reply:
Created https://github.com/KittyCAD/modeling-app/issues/4709

Reply:
```suggestion
        status: DEV || IS_NIGHTLY_OR_DEBUG ? 'available' : 'kcl-only',
```

Let's have it like the others now

Reply:
nice, ty!